medpy.core.exceptions.SubprocessError
=====================================

::: {.currentmodule}
medpy.core.exceptions
:::

::: {.autoexception}
SubprocessError
:::
